webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/NodeApi.scala

```diff
@@ -134,6 +150,9 @@ class NodeApi (
       case API.UpdateNode               => UpdateNode
       case API.ListAcceptedNodes        => ListAcceptedNodes
       case API.ApplyPolicy              => ApplyPolicy
+      case API.NodeDetailsTable         => NodeDetailsTable
```

Comment:
it should be before UpdateNode otherwise it will evaluate the "details" as a nodeid

Reply:
It's not here but in endpoints definition

Reply:
order here is not important

Reply:
i had to reorder that so often that i'm fairly sure it is